README.md

```diff
@@ -100,7 +121,25 @@ Each lion should have...
 Create a new lion instance with the name "Simba".
 
 ```ruby
-# code here
+class Animal
+  attr_reader :name, :greet
```

Comment:
` attr_accessor :name` is the short hand to have the ability to `get` and `set` a property.